src/index.spec.js

```diff
@@ -245,7 +245,7 @@ describe('hoc-react-loader', () => {
 
       const mounted = mount(<Wrapped some="props" />)
       mounted.unmount()
-      expect(mounted.html()).toBe(null)
+      expect(mounted.exists()).toBe(false)
```

Comment:
Maybe we could use [enzyme matchers](https://github.com/FormidableLabs/enzyme-matchers/blob/master/packages/jest-enzyme/README.md)

Reply:
Why not, I create an issue https://github.com/alakarteio/hoc-react-loader/issues/95 :)